### DISCUSSION TITLE
Exit Relay on Smartphone ?

### DISCUSSION THREAD

**Discussion by nitmeare** [Tribler/tribler#7580] [General]
I wonder if it is possible to set up an Tribler Exit relay on a old smartphone. This way it may be possible to use
some corporate WiFi Networks, as example in shopping Centers, Mc Donalds and so on as a temporary anonymous exit relay.

**Comment by qstokkink** [comment]
Technically, the answer would be "yes": you "only" need to implement/port the relevant code and run it on your device.

Practically speaking, though, I don't think anyone wants to spend the (significant amount of) time to port the Tribler code to an old device. To add to this, at this moment in time, the Tribler network also even has a surplus of exit nodes. So, realistically, my answer would be "no".
